batch/snippets/src/main/java/com/example/batch/CreateBatchRunnableLable.java

```diff
@@ -0,0 +1,138 @@
+import java.util.concurrent.TimeoutException;
+
+public class CreateBatchRunnableLable {
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Google Cloud project you want to use.
+    String projectId = "YOUR_PROJECT_ID";
+    // Name of the region you want to use to run the job. Regions that are
+    // available for Batch are listed on: https://cloud.google.com/batch/docs/get-started#locations
+    String region = "us-central1";
+    // The name of the job that will be created.
+    // It needs to be unique for each project and region pair.
+    String jobName = "example-job";
+    // Name of the label1 to be applied for your Job.
+    String labelName1 = "RUNNABLE_LABEL_NAME1";
+    // Value for the label1 to be applied for your Job.
+    String labelValue1 = "RUNNABLE_LABEL_VALUE1";
+    // Name of the label2 to be applied for your Job.
+    String labelName2 = "RUNNABLE_LABEL_NAME2";
+    // Value for the label2 to be applied for your Job.
+    String labelValue2 = "RUNNABLE_LABEL_VALUE2";
+
+    createBatchRunnableLable(projectId, region, jobName, labelName1,
+        labelValue1, labelName2, labelValue2);
+  }
+
+  // Creates a job with labels defined in the labels field
+  // for a runnable are only applied to that runnable
+  public static Job createBatchRunnableLable(String projectId, String region, String jobName,
+                   String labelName1, String labelValue1, String labelName2, String labelValue2)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (BatchServiceClient batchServiceClient = BatchServiceClient.create()) {
+
+      // Define what will be done as part of the job.
+      Runnable runnable =
+          Runnable.newBuilder()
+              .setContainer(
+                  Runnable.Container.newBuilder()
+                      .setImageUri("gcr.io/google-containers/busybox")
+                      .setEntrypoint("/bin/sh")
+                      .addCommands("-c")
+                      .addCommands(
+                          "echo Hello world! This is task ${BATCH_TASK_INDEX}. "
+                              + "This job has a total of ${BATCH_TASK_COUNT} tasks.")
+                      .build())
+              // Label and its value to be applied to the first runnable.
+              .putLabels(labelName1, labelValue1)
+              .setScript(Runnable.Script.newBuilder()
+              .setText("echo Hello world! This is task ${BATCH_TASK_INDEX}. ").build())
+              // Label and its value to be applied to the second runnable.
+              .putLabels(labelName2, labelValue2)
```

Comment:
IIUC, this is supposed to be part of the second runnable. The sample should demonstrate setting labels in two different runnables. The first runnable is a container job and the second one is script job. 
Kindly refer the doc here: https://cloud.google.com/batch/docs/organize-resources-using-labels

@m-strzelczyk Please confirm if this understanding is correct.